aws/resource_aws_cloudfront_distribution.go

```diff
@@ -478,6 +478,56 @@ func resourceAwsCloudFrontDistribution() *schema.Resource {
 					},
 				},
 			},
+			"origin_group": {
+				Type:     schema.TypeSet,
+				Optional: true,
+				Set:      originGroupHash,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"origin_id": {
+							Type:         schema.TypeString,
+							Required:     true,
+							ValidateFunc: validation.NoZeroValues,
+						},
+						"failover_criteria": {
+							Type:     schema.TypeSet,
```

Comment:
Can more than 1 failover criteria configuration be defined? I would presume not and would suggest changing this attribute to `TypeList` and `MaxItems: 1` along with removing `Set`.